python/setup.py

```diff
@@ -100,13 +111,26 @@ def config_cython():
         print("WARNING: Cython is not installed, will compile without cython module")
         return []
 
-setup(name='mxnet',
-      version=__version__,
+# Create a custom wheel class to ensure the wheel is limited to
+# the platforms we actually support
```

Comment:
Useful comment, but would it make sense to add a little clarification regarding platform support?  We're still planning to release binary wheels that aren't fully x86-64 compatible, but the comment reads to me like they all will be.